frontend/components/RedeemProductsSlider/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'glamor';
+import { I18n } from '@shopgate/engage/components';
+import { useTheme } from '@shopgate/engage/core';
+import connect from './connector';
+
+const headline = css({
+  fontSize: '22px',
+  margin: '12px 16px',
+});
+
+/**
+ * RedeemProductsSlider
+ * @param {Function} fetchProducts Fetch products
+ * @param {string[]} redeemProducts Swell product redemption option Id
+ * @return {JSX}
+ */
+const RedeemProductsSlider = ({ fetchProducts, redeemProductIds = [] }) => {
```

Comment:
no default values here please. since you have it already in defaultProps

Reply:
done